src/languages/scala.js

```diff
@@ -107,7 +109,7 @@ export default function(hljs) {
   const METHOD = {
     className: 'function',
     beginKeywords: 'def',
-    end: /[:={\[(\n;]/,
+    end: regex.lookahead(/[:={\[(\n;]/),
     excludeEnd: true,
```

Comment:
If you're using lookahead then you shouldn't need `excludeEnd` anymore, as there will be no match to exclude.

Reply:
Removed